client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -275,6 +280,27 @@ export class CheckoutThankYou extends Component<
 		}
 	}
 
+	/**
+	 * Determines whether the current checkout flow is for a redesign V2 purchase.
+	 * Used for gradually rolling out the redesign.
+	 *
+	 * @returns {boolean} True if the checkout flow is for a redesign V2 purchase, false otherwise.
+	 */
+	isRedesignV2 = () => {
```

Comment:
Will move this out to a separate file in another PR so we can write tests for this.

Reply:
Nice, that sounds good. Will it also lives inside `redesign-v2` folder?